### DISCUSSION TITLE
Promote Linux Mint as an economical, social, ecological alternative to planned obsolescence

### DISCUSSION THREAD

**Discussion by SebastJava** [linuxmint/discussions#412] [Ideas]
I am using an old HP EliteDesk 800 G2 SFF. That computer is more than 8 years old. Yet, it all works good and everything is ultra-fast, instantaneous, on the latest Linux Mint Cinnamon.

I also have Windows 10 on another drive, on that same computer. I use it for a few minutes every now and then, just to see how things would look on this platform.

I can't upgrade from Windows 10 to Windows 11 because... my computer is supposedly too old. And I have a feeling this Windows 10 will gradually slow down with each software upgrade... Before going dead.

Linux Mint can save millions of computers that would otherwise go to the landfill. We could emphasize the economic, social and ecological benefits. This could start with an addition on the main page on the [main website](https://linuxmint.com/). This OS would benefit from being known to anti-poverty groups and environmentalists.

**Comment by ElmNumaeStNo37** [comment]
Agree. However, to my knowledge Ubuntu had dropped support to 32-bit systems and by extension includes Mint. But Debian still supports 32-bit and by extension so too LMDE 6. I guess your proposition might be confusing to new comers as many of them in the Linux Mint forums install LM 21.3 Cinnamon at their potato machines expecting it to work like a champ and then got disappointed and then lash at Mint.

**Comment by Nyxiad** [comment]
I work for a small business that takes care of people who struggle to use their computers (mostly old people). We've postponed upgrading them to Windows 11 for as long as possible; these users would freak out over the UI change from 10 to 11 so if the UI is changing anyway, why not change to Mint instead? I've been working on convincing my bosses that Linux Mint could be a good alternative to Windows 11. They both are warming up to the idea. We build personal connections with our customers and they place a lot of trust in us so endorsements from us and other businesses could make a big difference.

Perhaps Linux Mint and the community should reach out to businesses like this to get more exposure with regular people who would never even consider that there could be anything other than Windows or Mac, let alone install Linux themselves.

**Comment by InfoLibre** [comment]
They are other Linux distributions to do that : [SparkyLinux MinimalGUI i686](https://sparkylinux.org/download/stable/) or [BunsenLabs Linux](https://www.bunsenlabs.org/). And for very old computers, [antiX](https://antixlinux.com/).

**Comment by InfoLibre** [comment]
It's not really Linux Mint role. You should participate to https://endof10.org.

**Reply by LinuxGuy-cyber** [reply]
How did you get to know about this website? No one is talking about it. Whatever the people there are doing, I suspect if there'd be any effect. For this website to reach the masses, it's only possible if **every** Linux distro starts promoting this on their _Main page._
